ion-schema/src/system.rs

```diff
@@ -543,6 +543,14 @@ impl TypeStore {
         self.imported_type_ids_by_name.values().cloned().collect()
     }
 
+    /// Provides the [`TypeId`] associated with given name if it exists in the [`TypeStore`] as an imported type;
+    /// Otherwise returns None
```

Comment:
Comment should mention built in types.

Reply:
I have modified this method to only include imported types. For built-in types, there is already `get_type_id_by_name` API available.